recipe-server/normandy/base/tests/test_middleware.py

```diff
@@ -21,3 +33,58 @@ def test_append_slash_redirects_with_cache_headers(self, rf, settings):
         assert res['Location'] == url + '/'
         cache_control = set(res['Cache-Control'].split(', '))
         assert cache_control == {'public', f'max-age={cache_time}'}
+
+
+class TestNormandySecurityMiddleware(object):
+
+    @pytest.fixture
+    def enable_ssl_redirect(self, settings):
```

Comment:
Why's this fixture on the class but the other one isn't?

Reply:
Even though both fixtures are only used in this class, this fixture applies only to `NormandySecurityMiddleware` testing, whereas the logger applies to the entire module under test.